.github/workflows/update-prettier.yml

```
@@ -0,0 +1,25 @@
+name: Update Prettier
+on:
+  push:
+    branches:
+      - "dependabot/npm_and_yarn/prettier-*"
+  workflow_dispatch: {}
+jobs:
+  update_prettier:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: "lts/*"
+          cache: npm
+      - run: npm ci
+      - run: npm run lint:fix
+      - uses: gr2m/create-or-update-pull-request-action@v1.x
+        env:
+          GITHUB_TOKEN: ${{ secrets.OCTOKITBOT_PAT }}
```

Hm I think this should have been 

```suggestion
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

when the repository is created outside of @octokit. Probably a bug in https://github.com/octokit/create-octokit-project.js/